src/libs/ReportUtils.js

```diff
@@ -1371,6 +1371,16 @@ function getNewMarkerReportActionID(report, sortedAndFilteredReportActions) {
         : '';
 }
 
+/**
+  * Replace code points > 127 with C escape sequences, and return the resulting string's overall length
+  * Used for compatibility with the backend auth validator for AddComment
+  * @param {String} textComment
+  * @returns {Number}
+  */
+function commentLength(textComment) {
```

Comment:
Can we rename this `getCommentLength` please?

Reply:
Absolutely